gamemode/languages/sh_spanish.lua

```diff
@@ -217,5 +280,157 @@ LANGUAGE = {
 	disable = "Deshabilitar",
 	vendorPriceReq = "Introduce el nuevo precio del objeto.",
 	vendorEditCurStock = "Editar Stock actual",
-	you = "Tú"
+	you = "Tú",
+	vendorSellScale = "Escala de precio de venta",
+	vendorNoTrade = "¡No puedes intercambiar con este vendedor!",
+	vendorNoMoney = "Este vendedor no puede permitirse comprar ese objeto.",
+	vendorNoStock = "Este vendedor no tiene ese objeto en Stock.",
+	contentTitle = "Falta el Contenido de Helix",
+	contentWarning = "No tienes el contenido de Helix montado. Esto puede resultar en que falten ciertas características.\n¿Quieres abrir la pagina de Workshop del contenido de Helix?",
+	flags = "Flags",
+	mapRestarting = "¡El mapa se reiniciara en %d segundos!",
+	chooseTip = "Escoge este personaje con el que jugar.",
+	deleteTip = "Eliminar este personaje.",
+	storageInUse = "¡Alguien ya esta usando esto!",
+	storageSearching = "Buscando...",
+	container = "Depósito",
+	containerPassword = "Has establecido la contraseña de éste depósito como %s.",
+	containerPasswordRemove = "Has eliminado la contraseña de éste depósito.",
+	containerPasswordWrite = "Introduce la contraseña.",
+	containerName = "Has cambiado el nombre de este depósitos a %s.",
+	containerNameWrite = "Introduce el nombre.",
+	containerNameRemove = "Has borrado el nombre de este depósito.",
+	containerInvalid = "¡Necesitas estar mirando a un depósito para hacer esto!",
+	wrongPassword = "Has introducido una contraseña errónea.",
+	respawning = "Reapareciendo...",
+
+	scoreboard = "Scoreboard",
+	ping = "Ping: %d",
+	viewProfile = "Ver el perfil de Steam.",
+	copySteamID = "Copiar Steam ID",
+
+	money = "Dinero",
+	moneyLeft = "Tu Dinero: ",
+	currentMoney = "Dinero Restante: ",
+
+	invalidClass = "¡Esa no es una clase valida!",
+	invalidClassFaction = "¡Esa no es una clase valida para la facción!",
+
+	miscellaneous = "Misceláneo",
+	general = "General",
+	observer = "Observador",
+	performance = "Rendimiento",
+	thirdperson = "Tercera Persona",
+	date = "Fecha",
+	interaction = "Interacción",
+	server = "Servidor",
+
+	resetDefault = "Reestablecer a predeterminado",
+	resetDefaultDescription = "Esto reestablecerá \"%s\" a su valor predeterminado de \"%s\".",
+	optOpenBags = "Abrir bolsas con inventario",
```

Comment:
This text is for literally opening bags, not items or objects.

Reply:
It can be a suitcase too, many inventory objects can contain stuff. In fact there was a suggestion on the forums to add a 1x1 inventory to even things like boots and such. That's why I changed it to "objects".

Reply:
Forgot there could be other items with inventory different than bags, so you're right. This would work not just for bags, but for any item that has an inventory.